src/ontology/edits/oeo-shared-axioms.omn

```diff
@@ -3673,20 +3673,9 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1034",
     
 Class: OEO_00020166
 
-    Annotations: 
-        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-shared",
-        <http://purl.obolibrary.org/obo/IAO_0000115> "A methodology is a plan specification that describes processes that are part of a study.",
-        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/971
-pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1011
-
-move to oeo-shared:
-issue: https://github.com/OpenEnergyPlatform/ontology/issues/1129
-pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1306"@en,
-        rdfs:label "methodology"
-    
     SubClassOf: 
-        <http://purl.obolibrary.org/obo/IAO_0000104>,
-        <http://purl.obolibrary.org/obo/BFO_0000050> some OEO_00020011
+        <http://purl.obolibrary.org/obo/IAO_0000104>
```

Comment:
`methodology SubClassOf: 'plan specification'` should be deleted here.

Reply:
[287ad28](https://github.com/OpenEnergyPlatform/ontology/pull/1673/commits/287ad286ec6aea5f947a701d3c7dcf984bdf8968)